### ISSUE TITLE
Not the correct size in the text input if I use the .form-group-lg in a horizontal form

### ISSUE THREAD

**Issue by Nikolay-Lipovtsev** [twbs/bootstrap-sass#823]
When I create a horizontal form with class size, the text input is generated with incorrect height. It is created with height 68px (the correct height for the Bootstrap official documentation is 46px).

This is my html:
```html
<form role="form" class="form-horizontal" id="new_user" action="/users" accept-charset="UTF-8" method="post">
  ...
  <div class="form-group form-group-lg">
    <label class="control-label col-sm-2" for="user_name">Control size</label>
    <div class="col-sm-10">
      <input class="form-control" type="text" name="user[name]" id="user_name">
    </div>
  </div>
</form>
```

I think this is happening because in CSS of `.form-group.form-group-lg > .form-control` remains `height: auto`;

This is my large input:
![2015-01-14 19 11 39](https://cloud.githubusercontent.com/assets/5441376/5752741/2d381156-9c8d-11e4-930a-7577b4896155.png)

This is normal large input:
![2015-01-14 19 12 18](https://cloud.githubusercontent.com/assets/5441376/5752744/370db82a-9c8d-11e4-8cf4-61f03b60a69d.png)

CSS:
![2015-01-15 8 06 33](https://cloud.githubusercontent.com/assets/5441376/5752764/85350f44-9c8d-11e4-9a87-87111ce46214.png)

When I use `http://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css` instead of bootstrap-sass it is ok.

I use:
Safari 8.0.2 (10600.2.5)

Gems:
rails 4.2.0
sass-rails 4.0.5
sass 3.2.19
bootstrap-sass 3.3.1.0

**Comment by glebm** [comment]
Possibly related: https://github.com/twbs/bootstrap/issues/15074 and #763

**Comment by glebm** [comment]
Confirmed with branch `next` output, and this is indeed the same issue as #763. Fixed in https://github.com/twbs/bootstrap-sass/commit/41ea576e9d611a9b2fffe4b1e0759ee7824858bc, available on branch `next`, to be released in v3.3.2.
